addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -458,4 +458,51 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
             color: $gray-600;
         }
     }
+    .o_survey_choice_image img {
+        cursor: default;
+        &:hover {
+            box-shadow: none;
+        }
+    }
+}
+
+/**********************************************************
+            Image (Zoom widget) Specific Style
+ **********************************************************/
+
+.o_survey_choice_image_modal {
+    .modal-dialog {
+        height: 100%;
+        max-width: 100%;
+        background-color: rgba(0,0,0,0.8);
+        padding-bottom: 0px;
+        padding-top: 0px;
+    }
+    .o_survey_slide {
+        display: flex;
+        align-items: center;
+        img {
+            width: 90%;
+            height: 100%;
+            margin: auto;
+            object-fit: contain;
+        }
+    }
 }
+
+.o_survey_choice_image img {
+    max-width: 95%;
+    max-height: 60vh;
+    cursor: zoom-in;
```

Comment:
I think you also need `cursor: -webkit-zoom-in;`. Doesn't solve the linux issue though.